src/app/pages/PerpetualPage/hooks/usePerpetual_OpenPositions.ts

```diff
@@ -45,7 +49,12 @@ export const usePerpetual_OpenPosition = (
     data: tradeEvents,
     previousData: previousTradeEvents,
     loading,
-  } = useGetTraderEvents([Event.TRADE], address.toLowerCase());
+  } = useGetTraderEvents(
+    [Event.TRADE],
+    address.toLowerCase(),
+    'blockTimestamp',
+    OrderDirection.desc,
+  );
```

Comment:
This is fine now, but could get us in trouble with passionate traders, once they have many trade-events, because we are fetching everything right now to be able to handle multiple pairs in the future,
```suggestion
  } = useGetTraderEvents(
    [Event.TRADE],
    address.toLowerCase(),
    'blockTimestamp',
    OrderDirection.desc,
  );
  // TODO: only query latest trade per pair, for performance reasons
```